app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -255,15 +300,28 @@ public void success(List<Trip> list) {
         });
     }
 
+    public void updateUserDestinationList() {
```

Comment:
Can you clarify what exactly happens when this method is invoked?

Reply:
It updates the local arrayList with any changes that have been made to the destinaion history (e.g deletes, creates).